packages/hebao_v1/contracts/modules/core/UpgraderModule.sol

```diff
@@ -48,13 +52,37 @@ contract UpgraderModule is BaseModule {
     {
     }
 
+    function upgradeWalletImplementation(address wallet)
+        external
+    {
+        require(msg.sender == address(this), "PROHIBITED");
+
```

Comment:
This means only this module can call it?

Reply:
yes